breathecode/certificate/views.py

```diff
@@ -287,10 +298,12 @@ def post(self, request, academy_id=None):
                 certs.append(cert)
             else:
                 raise ValidationException(
-                    'There is no certificate for this student and cohor',
+                    'There is no certificate for this student and cohort',
                     code=404,
                     slug='no-user-specialty')
-            generate_one_certificate.delay(cu.cohort_id, cu.user_id)
+            generate_one_certificate.delay(cu.cohort_id,
```

Comment:
Cover 287 `data=[]` and 260 `data={...}`

https://coveralls.io/builds/41576609/source?filename=breathecode%2Fcertificate%2Fviews.py